### ISSUE TITLE
Hellfire Torch not dopping (some times)

### ISSUE THREAD

**Issue by EnemyArea** [D2R-Reimagined/d2r-reimagined-mod#41]
Hey guys,

I want to inform that you maybe got the same issue with the Hellfire Torch as I'm. I killed the ueber bsoses and got an normal unique charm not a torch. after some research I found out, that the devs of diablo where lazy and just take a random unique charm from the unique items table. So it can be the torch or the Hellfire Torch, Peacemaker or Lifesever because they are all using the cm2 type. 

I hope it helps :)

**Comment by CollinHerber** [comment]
Looking into this

**Comment by CollinHerber** [comment]
Yea, I was able to basically second what you had said. I set the drop to be specifically the hellfire torch but it still seemed to drop a random one of the charms. This is also the case for other uniques as well, not just charms.
So unsure if this is something that can be resolved

**Comment by CollinHerber** [comment]
Closing this as unresolvable. Sorry.

**Comment by waltasan** [comment]
Allthough this is already closed, it would be great if this gets communicated better. I did my first torch run yesterday and was quite surprised AND annoyed that I did not get my desired hellfire torch. And it took me a lot of effort to get the charm, as I'm not an expert. So please let the people know that they get a charm when doing an uber run, but that the desired torch it not guaranteed.

**Comment by Delegus** [comment]
How many of bosses did you kill each time, how many times did you kill them, did you already have charms in your inventory or personal stash, are you playing on mod version 1.21.1 downloaded from NexusMods, do you have other mods installed via D2RMM alongside D2RR (which is not officially supported), did you pick up and ID all charms that dropped, can you provide video or save files so the problem can be replicated so it can be addressed faster?

**Comment by waltasan** [comment]
> How many of bosses did you kill each time, how many times did you kill them, did you already have charms in your inventory or personal stash, are you playing on mod version 1.21.1 downloaded from NexusMods, do you have other mods installed via D2RMM alongside D2RR (which is not officially supported), did you pick up and ID all charms that dropped, can you provide video or save files so the problem can be replicated so it can be addressed faster?
> 
> Issue #365 Issue #361 Issue #218

Thanks for your feedback.
This was my very first uber run in D2RR, there are no other mods installed on my computer.
I have had the unique charm that I found (Peacemaker) already in my inventory, so I had to return to town to put it in my shared stash tabs before I could pick it up. There dropped no other unique charm. And as it was my first uber run, I did not have any torches in my personal stash. My mod version is 1.21.1, even though 1.21.0 is displayed in the lower right corner when I start the game.
Unfortunately I don't have any video that I could provide, I'm just a casual player. And as CollinHerber already wrote above: "..it still seemed to drop a random one of the charms." (https://github.com/D2R-Reimagined/d2r-reimagined-mod/issues/41#issuecomment-2170405730)
The reason for my comment was not that I see this as a bug, but the possibility to find a charm other then a hellfire torch should be better communicated.
